packages/gatsby-source-wordpress/src/fetch.js

```diff
@@ -248,6 +248,37 @@ async function fetchData({
         }
       }
     }
+    // Gravity Forms
+    if (type == `wordpress__gf_forms`) {
+      for (let key in routeResponse) {
+        for (let formKey in routeResponse[key]) {
+          let form = routeResponse[key][formKey]
+          if (form && form.id && form.title) {
+            // Ex. /gf/v2/forms/12
+            let formUrl = `${url}/${form.id}`
+            let fetchedData = await fetchData({
+              route: { url: formUrl, type: `${type}` },
+              _verbose,
+              _perPage,
+              _hostingWPCOM,
+              _auth,
+              _accessToken,
+            })
```

Comment:
I think we would want to parallelize fetches because right now this would get 1 form data at a time which will slow bootstraping process if there are multiple forms. 

Take a look how we pararelize requests in https://github.com/kennethormandy/gatsby/blob/830da2ff03729acaa2267c7cec4110ffd5aeed61/packages/gatsby-source-wordpress/src/fetch.js#L374-L381 

(I see that you copied logic from menus fetching - that could also benefit from same treatment if possible)

Reply:
@pieh Thanks for the quick review! I’ve got an attempt at it here, but frankly I don’t think I know `async`/`await` well enough to finish it just yet. https://github.com/gatsbyjs/gatsby/commit/e1fbf74b1a80bf5784fff991f2306fe1389758ee

I’m also not sure if this approach is totally necessary though, is this outcome that you are looking for? That each property is requested via a URL rather than just doing one request?

```
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/title
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/description
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/labelPlacement
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/descriptionPlacement
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/button
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/fields
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/version
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/id
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/useCurrentUserAsAuthor
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/postContentTemplateEnabled
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/postTitleTemplateEnabled
=== [ Fetching wordpress__gf_forms ] === http://localhost:8080/wp-json/gf/v2/forms/1/postTitleTemplate
```

…rather than now, that does just one request to `/forms/1` and then gets all that info. This might be a side effect of me doing something else wrong though.

Reply:
I meant doing parallel requests for forms (not for individual properties of single form)

Reply:
Makes sense, that’s what I was aiming to do, but I think I overcomplicated the `async`/`await` in an array bit.